linux/install_ubuntu1604_nginx.sh

```diff
@@ -34,7 +34,7 @@ su - omero -c "OMEROVER=$OMEROVER ICEVER=$ICEVER bash -eux step04_all_omero.sh"
 
 su - omero -c "bash setup_omero_db.sh"
 
-OMEROVER=$OMEROVER bash -eux step05_ubuntu1404_nginx.sh
+OMEROVER=$OMEROVER bash -eux step05_ubuntu1406_nginx.sh
```

Comment:
`1604` no? Looks like this passed without failure leaving something ppa installed on 1604 which might point to a deeper issue.